03-Purpose-and-Principles/Documentation-Principles.md

```diff
@@ -0,0 +1,46 @@
+# Documentation Principles
+
+These principles are not set in stone -- if you'd like clarification, qualification, addition, removal, or changes to these principles, open an issue to discuss.
+
+## Documentation Scope
+
+- There are many types of applications that can be built with PureScript, each of which should have documentation. The group of people who write each type of application is best suited for managing its documentation.
+- All types of PureScript applications have pre-requisite knowledge, and the set of this knowledge which is common across all PureScript applications should be managed by the group of PureScript groups, an entity which we will name the PureScript organization.
+- The current scope of this project is the documentation managed by the PureScript organization. Other efforts can be responsible for documentation managed by PureScript groups.
+
+## Documentation Structure
+
+- Documentation is easiest to maintain if it follows a well-defined format. Templates offer a simple way of starting a new document.
```

Comment:
Yup! Any suggestions for what format the template should follow? Do you have any examples one could copy and paste and build upon from there?

Reply:
I think answering this question will be an important, huge next step, but I'm not sure how to answer that question at this time. :(

Actually, should we figure out these templates at a later stage? Perhaps the brainstorming one?

Reply:
Probably

Reply:
I created the following issue to track this: https://github.com/chexxor/purescript-documentation-discussion/issues/72